### PR TITLE
[UmatApplication] import mechanism + test

### PR DESCRIPTION
**Pull request by lluis-mv**
* Import the application as python module
* One test

### CODE REVIEW

File: applications/UmatApplication/tests/test_J2_plasticity.py [hunk 1]
```diff
@@ -0,0 +1,276 @@
+from __future__ import print_function, absolute_import, division
+#import KratosMultiphysics
+
+#import KratosMultiphysics.ConstitutiveModelsApplication as KratosMaterialModels
+#import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics
+from KratosMultiphysics import KratosUnittest
+
+from KratosMultiphysics import ConstitutiveModelsApplication
```
Comment (philbucher) [inline applications/UmatApplication/tests/test_J2_plasticity.py:9]: does this app also follow the new standard?

**Review by philbucher — DISMISSED** [review]
Nice :+1:

**Comment by philbucher** [comment]
@lluis-mv I recommend to take a look at codacy, it has many good suggestions

File: applications/UmatApplication/tests/test_J2_plasticity.py [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+        #material properties
+        self.properties = self.model_part.Properties[settings["properties_id"].GetInt()]
+
+        #read variables
+        self.variables = settings["variables"]
+        for key, value in self.variables.items():
+            variable = self._GetItemFromModule(key)
+            if( value.IsDouble() ):
+                self.properties.SetValue(variable, value.GetDouble())
+            elif( value.IsArray() ):
```
Comment (philbucher) [inline applications/UmatApplication/tests/test_J2_plasticity.py]:
```suggestion
            elif( value.IsVector() ):
```

File: applications/UmatApplication/tests/test_J2_plasticity.py (hunk 1 above)
Comment (lluis-mv) [reply applications/UmatApplication/tests/test_J2_plasticity.py:9]: not sure, i will talk with its maintainer

File: applications/UmatApplication/tests/test_J2_plasticity.py [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                vector_value = KratosMultiphysics.Vector(value.size())
+                for i in range(0, value.size() ):
+                    vector_value[i] = value[i].GetDouble()
+                self.properties.SetValue(variable, vector_value)
```
Comment (philbucher) [inline applications/UmatApplication/tests/test_J2_plasticity.py]:
```suggestion
                self.properties.SetValue(variable, value.GetVector())
```

Comment (lluis-mv) [reply applications/UmatApplication/tests/test_J2_plasticity.py]: I just erased that bit. I copy-pasted it from elsewhere but was not needed here.

**Review by philbucher — APPROVED** [review]

**Comment by lluis-mv** [comment]
thanks

**Pull request merged**